src/stores/auth.ts

```diff
@@ -82,6 +84,15 @@ const useAuthStore = defineStore('auth', {
         })
     },
 
+    authError(error: any) {
```

Comment:
一応、firebaseのauthにAuthErrorというinterfaceがあるみたいですが…
エラーは出ないけどこれでいいのかわからない…
https://firebase.google.com/docs/reference/js/v8/firebase.auth.AuthError
https://firebase.google.com/docs/reference/js/auth.autherror

```suggestion
    import { AuthError } from 'firebase/auth'

    authError(error: AuthError) {
```

Reply:
html の input タグに type を指定している && required にしています。
なので、新規登録のパスワードチェックしか validate をかけていません